examples/ipython/tutorial_algebra.ipynb

```diff
@@ -0,0 +1,672 @@
+{
```

Comment:
<p>Another option here would be to use something like</p><pre class="ql-syntax" spellcheck="false">display(
    (a+b).Fmt(title='a+b'),
    (a-b).Fmt(title='a-b'),
    (a*b).Fmt(title='ab'),
    (a|b).Fmt(title=r'a \cdot b'),
    ...
)
</pre><p>Which is perhaps a little more readable, but:</p><ul><li>Looks less pretty in the output</li><li>Does not appear in the <code>Out[n]</code>  cell any more.</li><li>Uses <code>Fmt</code>,  which we were trying not to mention till later</li><li>Only works on <code>mv</code>  objects, which means we end up having to switch between different notations</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pygae/galgebra/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>